### DISCUSSION TITLE
I wanted to buy Chrome license and was wrongly redirected to Firefox license

### DISCUSSION THREAD

**Discussion by rockyman2** [aclap-dev/video-downloadhelper#903] [License & Purchase requests]
### What's your issue?

I bought the license and apperently is intended to Firefox only. Can you guys deactivate Firefox and give me one for Chrome. Appreciate your support

### The 10 first digits (only 10) of your license number or email address

5b4a805156

### I confirm that I did not not paste my full license number

- [X] I confirm

**Comment by jcfrog** [comment]
hi,
sorry we can't swap licenses, what I can do is refund your Firefox premium so that you can purchase a Chrome premium if you want to.
Do you want me to do that?

**Reply by rockyman2** [reply]
that would be great. Really appreciate it. can you also include the link to buy the chrome one?

**Reply by jcfrog** [reply]
You have been refunded. Should be available in a few business days.
The links for each browser: https://github.com/aclap-dev/video-downloadhelper/wiki#purchase-premium
